### ISSUE TITLE
[DOC] Document new `EXPERIMENTAL_changes_in_block` RPC API

### ISSUE THREAD

**Issue by frol** [nearprotocol/docs#267]
As of nearprotocol/nearcore#2253 we have a brand new `EXPERIMENTAL_changes_in_block` API to fetch a list of accounts touched with some changes. We expose new APIs with a prefix `EXPERIMENTAL_`, so currently one can run a local node and use `EXPERIMENTAL_changes_in_block` endpoint to get the changes that are happening on the network on a per-block basis.

NOTE: This API is tightly related to `EXPERIMENTAL_changes` API #173 

## API

```
$ http post http://localhost:3030 jsonrpc=2.0 id=dontcare \
    method=EXPERIMENTAL_changes_in_block \
    'params:={"block_id": "DYHRzRbxUR1ANPdCQcgQE9g5zyYQoDZ1k8BJEQ3hDSgW"}'
```

```
$ http post http://localhost:3030 jsonrpc=2.0 id=dontcare \
    method=EXPERIMENTAL_changes_in_block \
    'params:={"block_id": 100500}'
```

```
$ http post http://localhost:3030 jsonrpc=2.0 id=dontcare \
    method=EXPERIMENTAL_changes_in_block \
    'params:={"finality": "latest"}'
```

NOTE: The endpoint naming needs some bikeshedding.

The response:

```js
{
    "block_hash": "DYHRzRbxUR1ANPdCQcgQE9g5zyYQoDZ1k8BJEQ3hDSgW",
    "changes": [
        {
            "type": "data_touched",
            "account_id": "frol.near"
        },
        {
            "type": "account_touched",
            "account_id": "frol.near"
        },
        {
            "type": "access_key_touched",
            "account_id": "user.near"
        },
    ]
}
```

**Comment by amgando** [comment]
@frol i'm unable to recreate this on staging.  am i missing something?

**Comment by amgando** [comment]
closed by #288

**Comment by amgando** [comment]
closed by #288
